test/node/api-versions.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application');
+
+const interfacesV1 = {
+  calculator: {
+    answer(connection, callback) {
+      callback(null, 42);
+    },
+  }
+};
+
+const interfacesV2 = {
+  calculator: {
+    answer(connection, callback) {
+      callback(null, 24);
+    },
+  }
+};
+
+const interfacesVLatest = {
```

Comment:
Wait... do I get it right that `latest` is a version by itself and not just a pointer to the latest semver version (`2.0.0` in this case)? That sounds wrong to me. `latest` and the like are tags, they must not be independent versions.